src/Uno.UI.RuntimeTests/IntegrationTests/dxaml/controls/grid/GridIntegrationTests.cs

```diff
@@ -464,9 +464,6 @@ public async Task CanSetCellPixelDimensions()
 			});
 		}
 
-#if __SKIA__
-		[Ignore("https://github.com/unoplatform/uno/issues/7271")]
-#endif
 		[TestMethod]
 		public async Task ValidateLayoutRoundingForPixelDimensions()
```

Comment:
In the issue, Jerome mentioned:

> For the ValidateLayoutRoundingForPixelDimensions is suspect this is related to DPI awareness.

@jeromelaban Is there something specific to change scaling on CI machines or something to see if the test will be still passing?

Reply:
At this time, there's no way to change the DPI of current window, but maybe having a helper to change the active value would help.

Reply:
As the test is passing, let's merge as-is.